__tests__/app.test.js

```diff
@@ -70,6 +70,51 @@ describe("/api/articles/:article_id", () => {
                 expect(response.body.msg).toBe("bad request");
             });
     });
+    test("PATCH:200 updates the given article votes in the database and responds with the updated article. the request body accepts an object with an inc_votes property which increments or decrements the vote count", () => {
+        return request(app)
+            .patch("/api/articles/5")
+            .send({ inc_votes: 10 })
+            .expect(200)
+            .then((addTenResponse) => {
+                return request(app)
+                    .patch("/api/articles/5")
+                    .send({ inc_votes: -5 })
+                    .expect(200)
+                    .then((subFiveResponse) => {
+                        expect(addTenResponse.body.article.votes).toBe(10);
+                        expect(subFiveResponse.body.article.votes).toBe(5);
+                    });
+            });
+    });
+    test("PATCH:404 responds with a suitable error when given a valid but non-existent article_id", () => {
+        return request(app)
+            .patch("/api/articles/101")
+            .send({ inc_votes: 25 })
+            .expect(404)
+            .then((response) => {
+                expect(response.body.msg).toBe("article_id not found");
+            });
+    });
+    test("PATCH:400 responds with a suitable error when given an invalid article_id", () => {
+        return request(app)
+            .patch("/api/articles/top-10-spider-facts")
+            .send({ inc_votes: 2 })
+            .expect(400)
+            .then((response) => {
+                expect(response.body.msg).toBe("bad request");
+            });
+    });
+    test("PATCH:400 responds with a suitable error when given an invalid newVotes object", () => {
+        return request(app)
+            .patch("/api/articles/top-10-spider-facts")
```

Comment:
You need to update this endpoint to something valid in order to test an invalid patch body.

Reply:
thanks good spot! - lesson learnt: don't copy and paste previous test code...